packages/cli/src/commands/package-dynamic-plugins/command.ts

```diff
@@ -143,6 +167,31 @@ COPY . .
     `,
       { cwd: tmpDir },
     );
+    Task.log(`Successfully built image ${tag} with following plugins:`);
+    for (const plugin of pluginRegistryMetadata) {
+      Task.log(`  ${chalk.white(Object.keys(plugin)[0])}`);
+    }
+    // print out a configuration example based on available plugin data
+    try {
+      const configurationExample = YAML.stringify({
+        plugins: pluginRegistryMetadata.map(plugin => {
+          const pluginName = Object.keys(plugin)[0];
```

Comment:
Just a very minor suggestion that can be ignored: I would use `packageName` for this variable name to keep consistency.

Reply:
yes, good idea, thanks!